src/models/language-id/LanguageIdJobOptions.ts

```diff
@@ -0,0 +1,14 @@
+import { CustomerUrlData } from '../CustomerUrlData';
+
+/**
+ * Options that can used when submitting Rev AI language id job.
+ * See https://docs.rev.ai/api/language-identification/reference/#operation/SubmitLanguageIdentificationJob for more details.
+ */
+export interface LanguageIdJobOptions {
+    media_url?: string;
```

Comment:
If this is deprecated, I don't think we should be adding it to new APIs. @aaron-wilson-rev what are your thoughts?

Reply:
Actually I think as long as we keep it in the documentation, we need it here. Since it could still get used

Reply:
Yeah I think we should have it here because the other sdks and our api still support it for language id. But if we add a new feature then we shouldn't continue adding it.